lms/routes.py

```diff
@@ -24,14 +24,15 @@ def includeme(config):
         "/content_item_selection",
         factory="lms.resources.LTILaunchResource",
     )
+
     config.add_route(
-        "canvas_oauth_callback",
-        "/canvas_oauth_callback",
+        "canvas_api.oauth.authorize",
+        "/api/canvas/oauth/authorize",
```

Comment:
I fixed the URL on this one, which we can do since this is a URL that our own app opens itself